README.md

```diff
@@ -331,6 +333,25 @@ $ glab config set editor vim --host gitlab.example.org
   NO_COLOR: set to any value to avoid printing ANSI escape sequences for color output. 
   ```
 
+### example
+
+on cli
+```sh
+export GITLAB_HOST=mygitlab.com 
+export GITLAB_API_HOST=api.mygitlab.com
+glab mr list
+```
+as config in global config file
+```yaml
+...
+hosts:
+    mygitlab.com:
+        api_host: api.mygitlab.com
+        token: mytoken
+...
+```
+
```

Comment:
Considering that we haven't added examples for the other env vars, the examples here won't be needed as it should be obvious to users on what to do.
The `api_host` can be set with `glab config set api_host api.mygitlab.com -h mygitlab.com`
```suggestion
```

I plan of modifying `glab auth login` so that user is prompted to set the `api_host`

Reply:
I added an example, because it was not that obvious. 

Even the command `glab config set api_host api.mygitlab.com -h mygitlab.com` which adds a host in the global config is not obvious besides the '--global' flag for configs. ;)

Reply:
While reading through the code I see some opportunities for refactorings:
- centralise config (I see glinstance.* invocation very deep down the invocation chain)
- maybe introduce a logger to enable debug logs overall the place

just my two cents

Reply:
> I added an example, because it was not that obvious.

Should be obvious as long as the `GITLAB_API_HOST` is indicated as part of the list of respected env vars and it is added to the config file well.

From my comment here: https://github.com/profclems/glab/pull/598#issuecomment-770300435
> I will open a follow-up PR with changes to glab auth login so that the user is prompted to set the api_host

`glab auth login` will have `--api-host` flag as well

Reply:
> - maybe introduce a logger to enable debug logs overall the place

Hmm... You're right on this! We can delve deeper into this in a separate issue/PR